packages/web-components/src/components/cbp-card/cbp-card.tsx

```diff
@@ -1,17 +1,37 @@
-import { Component, Host, h } from '@stencil/core';
+import { Component, Host, Element, Prop, h } from '@stencil/core';
+import { setCSSProps } from '../../utils/utils';
 
 @Component({
   tag: 'cbp-card',
   styleUrl: 'cbp-card.scss',
 })
 export class CbpCard {
+  @Element() host: HTMLElement;
+
+  @Prop({ reflect: true }) color: 'danger';
+  @Prop({ reflect: true }) variant: 'banner' | 'decision';
+  /** Supports adding inline styles as an object */
+  @Prop() sx: any = {};
+
+  componentWillLoad() {
+    if (typeof this.sx == 'string') {
+      this.sx = JSON.parse(this.sx) || {};
+    }
+    setCSSProps(this.host, {
+      ...this.sx,
+    });
+  }
 
   render() {
     return (
       <Host>
-        <slot></slot>
+        {this.variant === 'banner' && <slot name="cardtitle" />}
+        <div class="cardbody">
+          {this.variant !== 'banner' && <slot name="cardtitle" />}
```

Comment:
Nice solution for conditional placement of the card title. I'm not sure if it will behave if a prop is changed on the fly, but that should not be a real-world situation.